app/editor/src/components/layout/styled/Layout.tsx

```diff
@@ -33,14 +34,15 @@ export const Layout = styled.div`
 
   & > header {
     background-color: ${(props) => {
-      switch (process.env.NODE_ENV) {
-        case 'development':
-          return props.theme.css.developmentBackgroundColor;
-        case 'test':
-          return props.theme.css.testBackgroundColor;
-        default:
-          return props.theme.css.productionBackgroundColor;
-      }
+      const isDevelopment =
+        window.location.href.includes(env.environments[0]) ||
+        process.env.NODE_ENV === 'development';
+      const isTest =
+        window.location.href.includes(env.environments[1]) || process.env.NODE_ENV === 'test';
```

Comment:
`process.env.NODE_ENV` will always be `undefined` in a client application because it runs in the browser and not the server.  There is no value in having this logic.

Now you are hardcoding the array positions.  This is not maintainable.

Reply:
Updated. For process.env.NODE_ENV it still could be useful in our local environment, otherwise we will get the production background color unless we change the values in env.json. Thanks

Reply:
Think about the purpose of development imitating DEV/TEST/PROD.  If the behaviour is entirely different in your local environment then changing the background locally is pointless, because it won't behave that way in any other environment.  Do not add code that we have to maintain that does nothing outside of local development unless you have to.

Reply:
Also you can add a `localhost` value in the config

Reply:
Yeah just added it in, thanks